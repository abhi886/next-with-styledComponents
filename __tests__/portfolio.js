// __tests__/index.test.jsx

/**
 * @jest-environment jsdom
 */

import React from "react";
import { render, screen } from "@testing-library/react";
import Portfolio from "../pages/portfolio";

describe("Portfolio", () => {
  it("renders a heading", () => {
    render(<Portfolio />);

    const heading = screen.getByRole("heading", {
      name: /portfolio/i,
    });

    expect(heading).toBeInTheDocument();
  });
});
