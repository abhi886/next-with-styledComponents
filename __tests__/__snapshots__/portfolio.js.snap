// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div
  className="portfolio__PortfolioContainer-sc-u8dw1z-0 dkOTxm container"
>
  <div
    className="portfolio__PortfolioWrapper-sc-u8dw1z-1 iTieSi"
  >
    <h3
      className="portfolio__PortfolioTitle-sc-u8dw1z-2"
    >
       Portfolio:-) Content Coming Soon !!!! 
    </h3>
  </div>
</div>
`;
